databases/jdp/file.go

```diff
@@ -102,6 +100,9 @@ type Metadata struct {
 		// display name
 		DisplayName string `json:"display_name"`
 	} `json:"gold_data"`
+	IMG struct {
+		TaxonOID int `json:"taxon_oid"`
```

Comment:
I have been wondering what the "o" in "oid" stands for... any insight?

Reply:
Looks like it might be "object": https://img.jgi.doe.gov/object-identifiers.html